src/shared/helpers.ts

```diff
@@ -58,3 +58,25 @@ export const failure = (message: string) => ({
   success: false,
   message
 })
+
+interface ExchangeRates {
+  base: string
+  rates: Record<string, number>
+}
+
+export const getExchangeRates = async (): Promise<ExchangeRates> => {
+  const response = await fetch(
+    'https://telemetry-exchange-rates.s3.amazonaws.com/exchange-rates-usd.json'
```

Comment:
I think we have hardcoded url in a few places, maybe we should move them into an env/config file or

Reply:
Will move it in #231.